examples/tutorials/physics_benchmarking.py

```diff
@@ -0,0 +1,334 @@
+        sim.initialize_agent(
+            0,
+            habitat_sim.AgentState(
+                [5.45 * box_size, 2.4 * box_size, 1.2 * box_size],
+                np.quaternion(-0.83147, 0.2, -0.55557, -0.2),
+            ),
+        )
+
+        # get the physics object attributes manager
+        obj_templates_mgr = sim.get_object_template_manager()
+        # [/initialize]
+
+        # build box
+        cube_handle = obj_templates_mgr.get_template_handles("cube")[0]
+
+        box_parts = []
+        boxIDs = []
+        for _i in range(5):
+            box_parts += [obj_templates_mgr.get_template_by_handle(cube_handle)]
+        box_parts[0].scale = np.array([1.0, 0.1, 1.0]) * box_size
+        box_parts[1].scale = np.array([1.0, 0.6, 0.1]) * box_size
+        box_parts[2].scale = np.array([1.0, 0.6, 0.1]) * box_size
+        box_parts[3].scale = np.array([0.1, 0.6, 1.0]) * box_size
+        box_parts[4].scale = np.array([0.1, 0.6, 1.0]) * box_size
+
+        for i in range(5):
+            part_name = "box_part_" + str(i)
+            obj_templates_mgr.register_template(box_parts[i], part_name)
+            boxIDs += [sim.add_object_by_handle(part_name)]
+            sim.set_object_motion_type(
+                habitat_sim.physics.MotionType.KINEMATIC, boxIDs[i]
+            )
+        sim.set_translation(np.array([2.50, 0.05, 0.4]) * box_size, boxIDs[0])
+        sim.set_translation(np.array([2.50, 0.35, 1.30]) * box_size, boxIDs[1])
+        sim.set_translation(np.array([2.50, 0.35, -0.50]) * box_size, boxIDs[2])
+        sim.set_translation(np.array([3.40, 0.35, 0.4]) * box_size, boxIDs[3])
+        sim.set_translation(np.array([1.60, 0.35, 0.4]) * box_size, boxIDs[4])
+        for i in range(5):
+            sim.set_object_motion_type(habitat_sim.physics.MotionType.STATIC, boxIDs[i])
+
+        # load some object templates from configuration files
+        object_ids = []
+        scales = {
+            "test_assets/objects/sphere": 1,
+            "test_assets/objects/chair": 0.9,
+            "test_assets/objects/donut": 2,
+            "test_assets/objects/nested_box": 0.5,
+        }
+        for obj_path in objects:
+            object_ids += [
+                obj_templates_mgr.load_configs(str(os.path.join(data_path, obj_path)))[
+                    0
+                ]
+            ]
+            obj_template = obj_templates_mgr.get_template_by_ID(object_ids[-1])
+            if obj_path in scales.keys():
+                obj_template.scale *= scales[obj_path]
+            obj_templates_mgr.register_template(obj_template)
+
+        # throw objects into box
+        for i in range(num_objects):
+            cur_id = sim.add_object(object_ids[i % len(object_ids)])
+
+            obj_node = sim.get_object_scene_node(cur_id)
+            obj_bb = obj_node.cumulative_bb
+            diaganal_length = math.sqrt(
+                obj_bb.size_x() ** 2 + obj_bb.size_y() ** 2 + obj_bb.size_z() ** 2
+            )
```

Comment:
Typo spelling.
Also, Magnum utils for this:
```suggestion
            diagonal_length = obj_bb.size().length()
```